test/cli_test.py

```diff
@@ -79,15 +83,29 @@ def test_symlink_view(self):
                 'formats': 'link',
             }
         }
+        self.alt_config = self.config['alternatives']['by-year']
+
+    def test_add_move_remove_album(self):
+        self.add_album(artist='Michael Jackson', album='Thriller',
+                       year='1990', original_year='1982')
+
+        self.runcli('alt', 'update', 'by-year')
 
-        self.add_album(artist='Michael Jackson', album='Thriller', year='1982')
+        by_year_path = self.lib_path(b'by-year/1990/Thriller/track 1.mp3')
+        target_path = self.lib_path(b'Michael Jackson/Thriller/track 1.mp3')
+        self.assertSymlink(by_year_path, target_path)
 
+        self.alt_config['paths']['default'] = '$original_year/$album/$title'
         self.runcli('alt', 'update', 'by-year')
 
-        self.assertSymlink(
-            self.lib_path(b'by-year/1982/Thriller/track 1.mp3'),
-            self.lib_path(b'Michael Jackson/Thriller/track 1.mp3'),
-        )
+        by_orig_year_path = self.lib_path(b'by-year/1982/Thriller/track 1.mp3')
+        self.assertIsNotSymlink(by_year_path)
```

Comment:
I'm also not really happy with that name. I added an additional assertion, because `assertIsFile` uses `os.path.isfile`, which IIRC follows symlinks and stats the target, i.e. if for some reason a symlink ends up being broken, it will not be detected. So if read the documentation correctly, `os.path.lexists` should precisely check that the path does not exist at all. Which `os`/`os.path` method do you think should be used by this test?

Reply:
How about renaming `assertIsNotSymlink` to `assertIsNotFile` and use ` not os.path.isfile() and not os.path.islink()`?